burn-book/src/SUMMARY.md

```diff
@@ -13,16 +13,15 @@
   - [Tensor](./building-blocks/tensor.md)
   - [Autodiff](./building-blocks/autodiff.md)
   - [Module](./building-blocks/module.md)
-  - [Config]()
```

Comment:
I think we should keep a section on Config because it's central how modules can be created. It's a great development pattern. Also Burn's philosophy on saving configuration separately should be mentioned.